src/main/java/AddEventCommand.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Class to present Adding event command.
+ */
+public class AddEventCommand extends Command {
+
+    private Parser parser;
+
+    /**
+     * Constructor.
+     * @param inputCommand command inputted by user.
+     * @param isExit whether to exit the chat bot or not.
+     */
+    public AddEventCommand(String inputCommand, boolean isExit, Parser parser) {
+        super(inputCommand, isExit);
+        this.parser = parser;
+    }
+
+    /**
+     * Execute the given task.
+     * @param ui the Ui class in charge of aesthetics.
+     * @param list tasklist that stores all the task.
+     * @param storage storage that writes to file.
+     * @throws DukeException exception.
```

Comment:
It's better to specify the condition that will result in the exception, instead of just stating that it is an exception.